docs/user-guide/configure.md

```diff
@@ -648,3 +648,9 @@ For example:
 > This config option should not be overridden on a per-file basis.
 
 [More info](options.md#fix).
+
+## `formatter`
+
+Specify the formatter to format your results.
+
+[More info](options.md#formatter).
```

Comment:
[suggest] Can you add a sample config code for `formatter`? It should help people understand the usage.

Reply:
Done. Not sure about the format though.